### DISCUSSION TITLE
Multitenancy -> v0.26.0 ?

### DISCUSSION THREAD

**Discussion by knkcni** [actions/actions-runner-controller#1782] [Questions] [answered]
Hey guys, 

I wanted to implement [Multitenancy](https://github.com/actions-runner-controller/actions-runner-controller#multitenancy) in my cluster, however, documentation refer to v0.26.0, which is not released yet, not even in pre-release. Is this a typo or a forcast ?

> This feature requires controller version => [v0.26.0](https://github.com/actions-runner-controller/actions-runner-controller/releases/tag/v0.26.0)

**Comment by toast-gear** [comment, marked as answer]
It's coming soon
